examples/basics/batches.ipynb

```diff
@@ -200,52 +181,111 @@
+        "  priority=5 # priority between 1(Highest) - 5(lowest)\n",
+        ")\n",
+        "# number of data rows in the batch\n",
+        "print(\"Number of data rows in batch: \", batch.size)"
+      ],
+      "cell_type": "code",
+      "outputs": [],
+      "execution_count": null
+    },
+    {
+      "metadata": {},
+      "source": [
+        "### Create multiple batches\n",
+        "The `project.create_batches()` method accepts up to 1 million data rows.  Batches are chunked into groups of 100k if necessary, which is the maximum batch size. This method takes in a list of either data row IDs or global keys (but not both).\n",
+        "\n",
+        "In the code below, only one batch will be created, since we are only using the few data rows we created above. Creating over 100k data rows for this demonstration is not sensible, but this method is the preferred approach for batch creation as it will gracefull handle massive sets of data rows."
       ],
       "cell_type": "markdown"
     },
     {
       "metadata": {},
       "source": [
-        "batch = project.create_batch(\n",
-        "  \"Demo-First-Batch\", # Each batch in a project must have a unique name\n",
-        "  sample, # A list of data rows or data row ids\n",
-        "  5 # priority between 1(Highest) - 5(lowest)\n",
+        "# First, we must create a second project so that we can re-use the data rows we already created.\n",
+        "second_project = client.create_project(\n",
+        "  name=\"Second-Demo-Batches-Project\",                                 \n",
+        "  media_type=lb.MediaType.Image\n",
         ")\n",
-        "# number of data rows in the batch\n",
-        "print(\"Number of data rows in batch: \", batch.size)"
+        "print(\"Project Name: \", second_project.name, \"Project ID: \", second_project.uid)\n",
+        "\n",
+        "# Then, use the method that will create multiple batches if necessary.\n",
+        "task = second_project.create_batches(\n",
+        "  name_prefix=\"demo2\",\n",
+        "  data_rows=data_row_ids,\n",
+        "  priority=5\n",
+        ")\n",
+        "\n",
+        "print(\"Errors: \", task.errors())\n",
+        "print(\"Result: \", task.result())"
       ],
       "cell_type": "code",
-      "outputs": [
-        {
-          "name": "stdout",
-          "output_type": "stream",
-          "text": [
-            "Number of data rows in batch:  4\n"
-          ]
-        }
+      "outputs": [],
+      "execution_count": null
+    },
+    {
+      "metadata": {},
+      "source": [
+        "### Create batches from a dataset\n",
+        "\n",
+        "If you wish to create batches in a project using all the data rows of a dataset, instead of having to gather the global keys or data row IDs, you can use the `project.create_batches_from_dataset()` method. This method takes in a dataset ID and creates a batch or batches comprised of all data rows not already in the project."
```

Comment:
May I suggest the following update: "If you wish to create batches in a project using all the data rows of a dataset, instead of using subsets of data rows you can use the `project.create_batches_from_dataset()` method. This method takes in a dataset ID and creates a batch or batches comprised of all data rows not already in the project."